readme.md

```diff
@@ -99,13 +99,18 @@ console.log("IO is a side effect!");
 
 ## Idempotency
 
-> A function is said to be idempotent if it has no side-effects on multiple
-executions with the the same input parameters.
+> A function is said to be idempotent if there are no additional effects of calling it a second time with the same parameters.
```

Comment:
Idempotency is simpler for pure functions. Should I just presume functions
are pure in the definition?

On Tue, Apr 26, 2016, 2:25 AM hemanth.hm <notifications@github.com> wrote:

> In readme.md
> <https://github.com/hemanth/functional-programming-jargon/pull/13#discussion_r61054232>
> :
>
> > @@ -99,13 +99,18 @@ console.log("IO is a side effect!");
> >
> >  ## Idempotency
> >
> > -> A function is said to be idempotent if it has no side-effects on multiple
> > -executions with the the same input parameters.
> > +> A function is said to be idempotent if there are no additional effects of calling it a second time with the same parameters.
>
> Well, I just an other 🐜 in this universe.
>
> @jethrolarson <https://github.com/jethrolarson> You could review the
> file, check if we could add some more missing definitions, thanks!
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/hemanth/functional-programming-jargon/pull/13/files/1223ee2a52772b817d3e1d9a3731757f9e083bd9#r61054232>
>